config/jobs/kubernetes/sig-testing/kubetest-canaries.yaml

```diff
@@ -244,6 +245,8 @@ periodics:
       args:
       - --cluster=e2e-kops-gce-canary.k8s.local
       - --deployment=kops
+      - --kops-ssh-user=ubuntu
+      - --env=KUBE_SSH_USER=ubuntu
```

Comment:
I'm not sure this should be set on the GCE jobs (see https://github.com/kubernetes/test-infra/pull/17950#issuecomment-644726859 )

/lgtm cancel